src/components/ProjectCard.svelte

```diff
@@ -34,7 +32,7 @@
   <div class="mentor">
     <div class="avatar">
       {#each p.maintainers as m, i}
-        <img src={avatars[m]} class="t-{i}" alt="" />
+        <img src={`https://avatars.githubusercontent.com/${m}`} class="t-{i}" alt="" />
```

Comment:
Having an alt text for an image tag is always considered a good practice. Keeping this in mind it would be great if could add the username of the maintainer as a value for the alt attribute instead of an empty string.

Reply:
Added alt attribute of maintainer.